airbyte-workers/src/main/resources/entrypoints/sync/main.sh

```diff
@@ -13,7 +13,7 @@ else
   echo "Using existing AIRBYTE_ENTRYPOINT: $AIRBYTE_ENTRYPOINT"
 fi
 
-((eval "$AIRBYTE_ENTRYPOINT ARGS" 2> STDERR_PIPE_FILE > STDOUT_PIPE_FILE) OPTIONAL_STDIN) &
+((eval "$AIRBYTE_ENTRYPOINT ARGS" 2> >(tee STDERR_PIPE_FILE >&2) | tee STDOUT_PIPE_FILE | grep -v '"type":"RECORD"' --line-buffered) OPTIONAL_STDIN) &
```

Comment:
to make sure I understand, can you explain this grep?

Reply:
This grep is to filter out messages that are actual data (since source pods output both logs and data to stdout).
Curious though whether you think this is a sufficient filter, since it assumes the json message will not have any extra whitespace (for example `{"type": "record", etc...}` would not match) and I don't think the connector spec says anything about removing unnecessary whitespace from json output.